bin/kong

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env lua
+#!/usr/bin/env luajit
```

Comment:
But people are installing Kong from many different packages, as well as from source. I am more worried about source and development installation. It's worth to put a note in the Changelog.